web/src/gql/queries/user.ts

```diff
@@ -0,0 +1,116 @@
+import { gql } from "@apollo/client";
+
+export const SEARCH_USER = gql`
+  query searchUser($nameOrEmail: String!) {
+    searchUser(nameOrEmail: $nameOrEmail) {
+      id
+      name
+      email
+    }
+  }
+`;
+
+export const ME = gql`
+  query Me {
```

Comment:
Queries should start with GetXxxx

```suggestion
export const GET_ME = gql`
  query GetMe {
```